samples/MvcSandbox/MvcSandbox.csproj

```diff
@@ -15,6 +15,8 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" />
     <PackageReference Include="Microsoft.Extensions.Logging.Console" />
     <PackageReference Include="Microsoft.Extensions.Logging.Debug" />
+
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="1.0.0" />
```

Comment:
planning to check this in? that needs @Eilon to approve

Reply:
Nope. That's just to get an idea of what the API description looks like. I'll undo the sample changes before this goes in.